WcaOnRails/app/views/competitions/payment_setup.html.erb

```diff
@@ -28,8 +28,16 @@
   <% else %>
     <p>
       This competition is set up to accept payments through Stripe.
-      If you ever want to disconnect your account, you can do so from your Stripe account dashboard
+      If you want to disconnect your account, you will first do so from your Stripe account dashboard
```

Comment:
After reading the discussion under the PR: Is this really technically necessary? Or do we just _prefer_ if the competitor first disconnects their Stripe Dashboard?

In any case, this phrasing of "you will first do..." sounds a bit harsh (to my non-native ears). What about "you should first do so from your Stripe dashboard"?

Reply:
The order doesn't matter, but both should be done so a new account can be associated.

Reply:
I updated it based on your phrasing.